jest.config.js

```diff
@@ -33,8 +33,5 @@ module.exports = {
     'jest-localstorage-mock',
   ],
   testURL: 'http://localhost',
-  moduleNameMapper: {
-    '^react-native$': 'react-native-web',
-    '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
-  },
+  testEnvironment: 'jsdom',
```

Comment:
i believe the default jest environment changed to be default node at some point https://jestjs.io/docs/configuration#testenvironment-string so it is now just jsdom by default